pydm/data_plugins/epics_plugins/p4p_plugin_component.py

```diff
@@ -175,8 +177,42 @@ def send_new_value(self, value: Value) -> None:
                     self._timestamp = value.timeStamp.secondsPastEpoch
                     self.timestamp_signal.emit(value.timeStamp.secondsPastEpoch)
 
+    @staticmethod
+    def convert_epics_nttable(epics_struct):
```

Comment:
A docstring for this and the method below would be helpful, just a sentence or two is fine